src/test/java/seedu/address/logic/commands/DeleteCommandTest.java

```diff
@@ -40,6 +47,53 @@ public void execute_deleteValidName_success() {
         assertCommandSuccess(deleteCommand, model, expectedMessage, expectedModel);
     }
 
+    @Test
+    public void execute_deleteValidNameUnfilteredList_success() {
+
+        ModelManager expectedModel = new ModelManager(model.getAddressBook(), new UserPrefs());
+
+        //filter the typical person list in the addressbook to only display Alice
+        model.updateFilteredPersonList(person -> person.isSamePerson(new Person(new FriendName("Alice Pauline"))));
+        expectedModel.updateFilteredPersonList(person -> person.isSamePerson(new Person(new FriendName("Alice Pauline"))));
+
+        assert(model.getFilteredPersonList().size() == 1);
+        assert(expectedModel.getFilteredPersonList().size() == 1);
+
+        Person personToDelete = model.getAddressBook().getPersonList().get(3);
+
+        //ensures that person to delete is not in the filtered person list
+        assert(!personToDelete.hasSameName(new Person(new FriendName("Alice Pauline"))));
+
+        //deletes the person that is not in the filtered person list
+        DeleteCommand deleteCommand = new DeleteCommand(personToDelete.getName());
+
+        String expectedMessage = String.format(DeleteCommand.MESSAGE_DELETE_PERSON_SUCCESS, personToDelete);
+
+        expectedModel.deletePerson(personToDelete);
+
+        assertCommandSuccess(deleteCommand, model, expectedMessage, expectedModel);
+    }
+
+    @Test
+    public void execute_deleteValidNameDifferentCase_success() {
```

Comment:
Low priority, but can consider delete valid name but for both filtered and unfiltered model states.

Reply:
Yup its done in other tests! this test is to test for names with different capitalisation